.github/workflows/go.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.20.x, 1.21.x, 1.22.x]
+        go-version: [1.22.x, 1.23.x, 1.24.x]
```

Comment:
Nit: there are `"stable"` and `"oldstable"` (however, testing 3 last versions will require a specific version) https://github.com/actions/setup-go#using-stableoldstable-aliases

Reply:
Theoretically `"stable"` can be used in `build-special` job below.

Reply:
Nice!